Vagrantfile

```diff
@@ -61,6 +61,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # Make the project root available to the guest VM.
   # config.vm.synced_folder '.', '/vagrant'
+  config.vm.synced_folder '.', '/vagrant'
+  config.vm.synced_folder '~/vagrantResources/', '/published'
```

Comment:
We can't include these options by default because it will be problematic for certain environments (Windows in particular).